python/lib/sift_py/_internal/channel.py

```diff
@@ -1,5 +1,42 @@
-from typing import Optional
+from typing import List, Optional, cast
+
+from sift.channels.v2.channels_pb2 import Channel as ChannelPb
+from sift.channels.v2.channels_pb2 import ListChannelsRequest, ListChannelsResponse
+from sift.channels.v2.channels_pb2_grpc import ChannelServiceStub
 
 
 def channel_fqn(name: str, component: Optional[str]) -> str:
     return name if component is None or len(component) == 0 else f"{component}.{name}"
+
+
+def get_asset_channels(
```

Comment:
Decided to move this to a more general location since this utility isn't specific to ingestion anymore.